starknet_py/common.py

```diff
@@ -2,8 +2,8 @@
 from typing import List, Literal, Optional, Union, cast
 
 from starknet_py.compile.compiler import Compiler, StarknetCompilationSource
-from starknet_py.net.client_models import ContractClass
-from starknet_py.net.schemas.gateway import ContractClassSchema
+from starknet_py.net.client_models import CompiledContract, ContractClass
+from starknet_py.net.schemas.gateway import CompiledContractSchema, ContractClassSchema
 
 
 def create_compiled_contract(
```

Comment:
After this gets removed, `_create_compiled_contract` would be ranamed to `create_compiled_contract` as it's just the perfect name, that is currently taken

Reply:
It will be problematic to properly deprecate `create_compile_contract` and then suddenly reintroduce it with different parameters.

Reply:
Actually I'd consider an alternative approach, but that would be for a separate bigger issue. 

What if we started introducing methods like this as `@classmethod`s in the models themselves.
```py
CompiledContract:
  ...

  @classmethod
  def from_contract(compiled_contract: str) -> CompiledContract:
    ...
```
(though I'm unsure about the exact name of the method).